code/datums/diseases/advance/symptoms/damage_converter.dm

```diff
@@ -42,6 +42,10 @@ Bonus
 
 		var/list/parts = H.get_damaged_organs(1,1) //1,1 because it needs inputs.
```

Comment:
probably much nicer to add an argument to ```get_damaged_organs()``` that allows you to specify organic only, robotic only, etc.